app/javascript/modules/listingDetailsLottery/ListingDetailsLotterySearchFooter.tsx

```diff
@@ -22,21 +22,31 @@ export const ListingDetailsLotterySearchFooter = ({
       </div>
       {lotterySearchFormStatus === LOTTERY_SEARCH_FORM_STATUS.INITIAL_STATE && (
         <div className="p-4 text-center text-tiny">
-          <p className="py-2">
-            {t("lottery.resultsFrom", {
-              date: localizedFormat(listing.Lottery_Results_Date, "LL"),
-            })}
-          </p>
-          <p className="py-2">
-            <a href={listing.LotteryResultsURL} target="_blank">
-              {t("lottery.lotteryResultsPdfLink")}
-            </a>
-          </p>
-          <p className="text-gray-700 py-2">
-            {t("lottery.unsortedResultsNote", {
-              date: localizedFormat(listing.Lottery_Date, "LL"),
-            })}
-          </p>
+          {listing.LotteryResultsURL ? (
+            <>
+              <p className="py-2">
+                {t("lottery.resultsFrom", {
+                  date: localizedFormat(listing.Lottery_Results_Date, "LL"),
+                })}
+              </p>
+              <p className="py-2">
+                <a href={listing.LotteryResultsURL} target="_blank">
+                  {t("lottery.lotteryResultsPdfLink")}
+                </a>
+              </p>
+              <p className="text-gray-700 py-2">
+                {t("lottery.unsortedResultsNote", {
+                  date: localizedFormat(listing.Lottery_Date, "LL"),
+                })}
+              </p>
+            </>
+          ) : (
+            <p className="py-2">
+              {t("lottery.resultsWillBePostedOn", {
+                date: localizedFormat(listing.Lottery_Results_Date, "LL"),
+              })}
+            </p>
```

Comment:
Can we rely on the Lottery_Results_Date being populated? Should we add a check just in case?

Reply:
let's do it!

Reply:
@christine-sfg pushed the update